src/rct/manifest_commands.py

```diff
@@ -206,6 +206,12 @@ def _print_consumer(self, zip_archive):
         to_print = []
         to_print.append((_("Name"), get_value(data, "name")))
         to_print.append((_("UUID"), get_value(data, "uuid")))
+        # contentAccessMode is entitlement if null, blank or non-present
+        contentAccessMode = 'entitlement'
+        if "contentAccessMode" in data:
+            if data["contentAccessMode"] == 'org_environment':
```

Comment:
Can you join these two with an `and`?  That'll make the code a little tidier.  Otherwise looks fine, and I'll merge after that change.